bitnami/zookeeper/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: zookeeper
-version: 4.3.4
+version: 4.4.4
```

Comment:
We should be setting this as a major version (`5.0.0`) due to there being a breaking change (the entrypoint is not compatible anymore).